src/services/storage/s3Service.test.ts

```
@@ -0,0 +1,109 @@
+  ALL_ALLOWED_MIME_TYPES,
+} from "./s3Service";
+
+// ── buildObjectKey ────────────────────────────────────────────────────────────
+
+describe("buildObjectKey", () => {
+  it("produces the expected kyc/{userId}/{kind}/{docId} pattern", () => {
+    const key = buildObjectKey(
+      "abc123",
+      "passport",
+      "doc-456",
+    );
```

ESLint

[error] Replace `⏎······"abc123",⏎······"passport",⏎······"doc-456",⏎····` with `"abc123",·"passport",·"doc-456"`

(prettier/prettier)